src/components/CustomClient.js

```diff
@@ -1,45 +1,66 @@
 import React from 'react';
-import { Divider } from 'semantic-ui-react';
+import { Divider, Form, Message } from 'semantic-ui-react';
 
-// Reference
-// https://codepen.io/hartzis/pen/VvNGZP?editors=0010
-// https://developer.mozilla.org/en-US/docs/Web/API/FileReader
+class CustomClient extends React.Component {
+  state = {
+    client: null,
+    error: null,
+    json: ''
+  };
```

Comment:
Setting the initial state should happen in a constructor, right?

Reply:
This is taking place in the constructor. I think the feature is called class fields, and I believe it is still just a proposal.

Reply:
Ah OK cool, I wasn't aware of that feature!